SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -79,10 +79,10 @@ FSpatialLoadBalancingHandler::EvaluateActorResult FSpatialLoadBalancingHandler::
 		}
 
 		const bool bNetOwnerHasAuth = NetOwner->HasAuthority();
+		const bool bShouldHaveAuthority = NetDriver->LoadBalanceStrategy->ShouldHaveAuthority(*NetOwner);
 
 		// Load balance if we are not supposed to be on this worker, or if we are separated from our owner.
-		if ((!NetDriver->LoadBalanceStrategy->ShouldHaveAuthority(*NetOwner) || !bNetOwnerHasAuth)
-			&& !NetDriver->LockingPolicy->IsLocked(Actor))
+		if ((!bShouldHaveAuthority || !bNetOwnerHasAuth) && !NetDriver->LockingPolicy->IsLocked(Actor))
```

Comment:
Is this easier to read? 
```suggestion
		if (!(bShouldHaveAuthority  && bNetOwnerHasAuth) && !NetDriver->LockingPolicy->IsLocked(Actor))
```